node-tests/unit/commands/exam-test.js

```diff
@@ -61,6 +61,12 @@ describe('ExamCommand', function() {
       });
     });
 
+    it('should set \'load-balance\' on the query option with one partition', function() {
```

Comment:
what does the `with one partition` mean? I'm not able to understand it looking at the test